vnext/ReactUWP/Utils/ResourceBrushUtils.cpp

```diff
@@ -65,7 +65,14 @@ void UpdateTextControlForegroundResourceBrushes(const winrt::FrameworkElement el
   UpdateResourceBrush(element, c_textControlButtonBackgroundPressed, brush);
 }
 
-void UpdateTextControlBorderResourceBrushes(const winrt::FrameworkElement &element, const winrt::Brush brush) {
+void UpdateTextControlBorderResourceBrushes(const winrt::FrameworkElement &element, const winrt::Brush b) {
+  // Workaround for bug https://microsoft.visualstudio.com/OS/_workitems/edit/26118890.
+  // Remove when the bug gets fixed.
+  winrt::Brush brush = b;
+  if (auto solidBrush = b.as<winrt::SolidColorBrush>()) {
+    brush = winrt::SolidColorBrush{solidBrush.Color()};
+  }
```

Comment:
IIUC we could still crash if someone happened to share some other type of brush between the border and the foreground, but that seems very unlikely (because the foreground is rarely ever anything other than a solid color).  If I've got this correct, then this fix seems like a fine compromise while we wait for a platform fix.  Thanks!